python/ray/train/tests/test_data_parallel_trainer.py

```diff
@@ -188,30 +215,6 @@ def train_func(config):
     assert trainer._train_loop_config["x"] == 100
 
 
-def test_scaling_config_validation(ray_start_4_cpus):
-    def train_func(config):
-        train.report({"loss": config["x"]})
-
-    # Should be able to create a DataParallelTrainer w/o scaling_config,
-    # but it should fail on fit
```

Comment:
We are able to launch training w/o scaling_config since `num_workers` defaults to 1 now.

Reply:
We used to force users to think about `num_workers`, but now we'll default to 1 silently -- think this is a UX problem?

We do show scaling config in our docs a lot so I think it should be fine.

Reply:
Some of our internal tests are using empty ScalingConfig.  
For the users I think it's fine. Our docstring and examples explicitly set num_workers.